Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ruby:2.6.5-stretch as basic
```

Comment:
> I believe that the `Dockerfile` in `.devcontainer` was auto generated and shouldn't be edited as it will likely cause the remote dev addon to fail. And after thinking about it, I don't think having both `Dockerfile`s is superfluous since they cater to different envs (which are essentially the same).

The `Dockerfile` was setup originally, but I've heavily edited. I'd make a go at using it for the dev environment before adding another.